### DISCUSSION TITLE
Un-install bluez-alsa

### DISCUSSION THREAD

**Discussion by seamusdemora** [Arkq/bluez-alsa#724] [Q&A] [answered]
I have two questions:

1. Is [this package](https://packages.debian.org/stable/bluez-alsa-utils) (`bluez-alsa-utils`) described on the Debian website essentially your repo - in "package form"? 

2. Your documentation in the wiki covers installation very well.  Is there a written procedure for **uninstalling** `bluez-alsa` if built and installed from your source repo?

**Comment by borine** [comment, marked as answer]
> Is [this package](https://packages.debian.org/stable/bluez-alsa-utils) (bluez-alsa-utils) described on the Debian website essentially your repo - in "package form"?

That is a question for the debian package maintainer to answer. Some distributions apply patches, others do not; each distribution has its own guidelines for selecting configure options and dependencies. So for any given distribution you would have to look at the package (or ask the package maintainers) to know exactly what it contains and how it was built.

> Your documentation in the wiki covers installation very well. Is there a written procedure for uninstalling bluez-alsa if built and installed from your source repo?

No specific documentation that I know of.  This question has never been asked before, and I have never uninstalled BlueALSA myself.  As BlueALSA uses the `autotools` build system,  its Makefile incudes a target called `uninstall`; so invoking `sudo make uninstall` in the build directory is supposed to remove all files installed by `sudo make install`. I've never tried it myself though, so I am offering no guarantees here.

**Comment by seamusdemora** [comment]
OK - I thought I'd try the **package**, but it occurred to me that I should probably uninstall the version I got here before doing that. My thinking was that perhaps the Debian folks might have the package set up/configured in such a way that it would address the too-frequent ***drop-outs*** I am seeing. I define a ***drop-out*** as when the sound suddenly & randomly stops after playing fine for some time (hours typically). The sound player (`mpg123` in my case) continues to run; oblivious to the fact that no sound is coming from the speaker. A `reboot` is required to recover. 

Anyway... I'll try the `uninstall` target - thanks for that!
